src/views/project-detail/TabCreateIssue.tsx

```diff
@@ -0,0 +1,131 @@
+// ** Next Imports
+import { useRouter } from 'next/router'
+
+// ** MUI Imports
+import Card from '@mui/material/Card';
+import CardHeader from '@mui/material/CardHeader';
+import CardContent from '@mui/material/CardContent';
+import { styled } from '@mui/material/styles'
+import Box, { BoxProps } from '@mui/material/Box';
+import Grid from '@mui/material/Grid'
+import IconButton from '@mui/material/IconButton'
+import Button, { ButtonProps } from '@mui/material/Button'
+import Avatar from '@mui/material/Avatar'
+import ArrowLeft from 'mdi-material-ui/ArrowLeft'
+
+// ** Custom Components Imports
+import CategoryTag, { Category } from 'src/views/project-detail/CategoryTag'
+import CreateIssueTag from './CreateIssueTag';
+import StatusTag, { Status } from "./StatusTag"
+
+import { useRecoilState } from 'recoil';
+import { activeView } from '../../recoil/issue/atom';
+
+const createReleaseNoteData = (releaseNoteId: number, version: string, releaseTitle: string, changeItems: ReleaseContent[], creatorId: string, creationDate: string) => {
+    return { releaseNoteId, version, releaseTitle, changeItems, creatorId, creationDate }
+}
+
+interface ReleaseContent {
+    category: Category,
+    releaseContent: string
+}
+
+interface DataType {
```

Comment:
```suggestion
interface ProjectDataType {
```

이렇게 하면 더 명확하지 않을까요~?!!

Reply:
오홍!! 좋아용 ㅎ👏🏻